test/v2/resources/projects/listProjects.test.js

```diff
@@ -12,14 +12,21 @@ import wrongParamsTests from '#test/v2/shared/wrongParams';
 
 jest.mock('axios');
 const client = tickspot({ apiVersion: 2, ...userInfo });
-const URL = `${client.baseURL}/projects.json`;
 
-describe('#listOpened', () => {
+const methods = ['listOpened', 'listClosed'];
+
+const getUrl = (method) => (method === 'listOpened'
+  ? `${client.baseURL}/projects.json`
+  : `${client.baseURL}/projects/closed.json`);
+
+describe.each(methods)('Project Methods', (method) => {
```

Comment:
@MrJuanGaviriaK What if we change it to the following?
```suggestion
describe.each(methods)('%s', (method) => {
```
This way tests will be grouped for each method. What do you think?

Reply:
Yes, I agree, thanks!!!